### PR TITLE
Fix The Game 🎉

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# TODO

- [x] Give Boxers no shirt/socks
- [x] Bring back the good roundend sounds (currently I chose a couple of the new ones that I thought were *okay*)

# Changelog

:cl:
- fix: Prescription glasses won't be given to people who aren't nearsighted
- fix: Cabbages and Science crates won't sell for more than they cost
- fix: Chargers won't give you the wrong circuit board when deconstructed
- fix: Power outage will send an announcement when complete
- fix: Shadowkin plushies are no longer pacified
- fix: Happy Honk boxes should have items in them now
- fix: Spells make you speak again
- fix: Chemistry guidebookbook works
- fix: Bluespace crystals use item space properly
- fix: Round end announcements work

### CODE REVIEW

**Comment by DEATHB4DEFEAT** [comment]
fake test fails jsndefjksdggskj

File: Content.Server/StationEvents/Events/PowerGridCheckRule.cs
Comment (Pspritechologist) [inline Content.Server/StationEvents/Events/PowerGridCheckRule.cs:1]: This file should be annotated as Parkstation changes.

File: Content.Shared/Roles/StartingGearPrototype.cs [hunk 1]
```diff
@@ -82,17 +82,17 @@ public string GetGear(string slot, HumanoidCharacterProfile? profile)
...
 
-                if (slot == "undershirt" && profile.Clothing != ClothingPreference.Jumpskirt && !string.IsNullOrEmpty(_undershirt) && _undershirt != "empty")
+                if (slot == "undershirt" && profile.Clothing != ClothingPreference.Jumpskirt && !string.IsNullOrEmpty(_undershirt) && _undershirt != "Error")
                     return _undershirt;
-                if (slot == "undershirt" && profile.Clothing == ClothingPreference.Jumpskirt && !string.IsNullOrEmpty(_undershirtskirt) && _undershirtskirt != "empty")
+                if (slot == "undershirt" && profile.Clothing == ClothingPreference.Jumpskirt && !string.IsNullOrEmpty(_undershirtskirt) && _undershirtskirt != "Error")
                     return _undershirtskirt;
 
-                if (slot == "socks" && !string.IsNullOrEmpty(_undersocks) && _undersocks != "empty")
+                if (slot == "socks" && !string.IsNullOrEmpty(_undersocks) && _undersocks != "Error")
```
Comment (Pspritechologist) [inline Content.Shared/Roles/StartingGearPrototype.cs]: "Error" is unintuitive to anyone reading the file :P

File: Resources/Prototypes/Roles/Jobs/Wildcards/boxer.yml [hunk 2]
```diff
@@ -34,6 +34,6 @@
   duffelbag: ClothingBackpackDuffelFilled
   underpants: ClothingUnderboxers_assblastusa
   underpantsskirt: ClothingUnderboxers_assblastusa
-  undershirt: empty
+  undershirt: Error
   undershirtskirt: ClothingUnderbra_binder
-  undersocks: empty
+  undersocks: Error
```
Comment (Pspritechologist) [inline Resources/Prototypes/Roles/Jobs/Wildcards/boxer.yml]: Ditto here

File: Resources/Prototypes/SimpleStation14/Entities/Objects/Devices/Circuitboards/siliconchargers.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@
       prototype: SiliconChargerChargePad
       requirements:
         Capacitor: 3
-        Laser: 5
+        Manipulator: 5
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Objects/Devices/Circuitboards/siliconchargers.yml:31]: Why's this changing?

File: Resources/Prototypes/SimpleStation14/Entities/Structures/Machines/silicon_chargers.yml [hunk 4]
```diff
@@ -119,22 +116,21 @@
...
+            - SlipLayer
+        fix2:
+          shape: !type:PhysShapeAabb
+            bounds: "-0.35,-0.35,0.35,0.35"
+          density: 10
+          mask:
+            - MachineMask
     - type: StepTrigger
     - type: SiliconCharger
-      chargeMylti: 150
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Structures/Machines/silicon_chargers.yml:137]: Are you kidding me :T

File: Resources/ServerInfo/SimpleStation14/Guidebook/AlertProcedure/AlertProcedure.xml [hunk 5]
```diff
@@ -4,12 +4,4 @@
...
 There are various possible alert levels, including some only activated by Central Command.
 Each alert level has different "Operating Procedures" that you must follow, shown in detail in the subcategories of this page.
-
-<Box HorizontalAlignment="Left">
-Alert levels are set either via a communications console, or the AI internal computer.
-</Box>
-<Box HorizontalAlignment="Left">
-<GuideEntityEmbed Entity="ComputerComms" Caption="" />
-<GuideEntityEmbed Entity="AICore" Caption="" />
-</Box>
```
Comment (Pspritechologist) [inline Resources/ServerInfo/SimpleStation14/Guidebook/AlertProcedure/AlertProcedure.xml:14]: Any reason for this?

File: Resources/Textures/Clothing/Head/Helmets/syndicate.rsi/meta.json
Comment (Pspritechologist) [inline Resources/Textures/Clothing/Head/Helmets/syndicate.rsi/meta.json:1]: I'd much rather avoid adding meta files for weird RSIs we don't understand. Can we simply delete the folder and see if it matters?
It wouldn't work at all without the meta, right?

File: Resources/Textures/Clothing/OuterClothing/Suits/syndicate.rsi/meta.json
Comment (Pspritechologist) [inline Resources/Textures/Clothing/OuterClothing/Suits/syndicate.rsi/meta.json:1]: Ditto

File: Resources/Textures/Clothing/OuterClothing/Vests/kevlar.rsi/meta.json
Comment (Pspritechologist) [inline Resources/Textures/Clothing/OuterClothing/Vests/kevlar.rsi/meta.json:1]: Ditto

File: Resources/Textures/Clothing/Uniforms/Jumpsuit/cluwne.rsi/meta.json
Comment (Pspritechologist) [inline Resources/Textures/Clothing/Uniforms/Jumpsuit/cluwne.rsi/meta.json:1]: Ditto

File: Resources/Textures/Corvax/Structures/Machines/VendingMachines/ammo.rsi/meta.json
Comment (Pspritechologist) [inline Resources/Textures/Corvax/Structures/Machines/VendingMachines/ammo.rsi/meta.json:1]: Ditto?

File: Resources/Prototypes/SoundCollections/announcements.yml [hunk 6]
```diff
@@ -3,9 +3,8 @@
   files:
   - /Audio/Announcements/RoundEnd/apc_destroyed.ogg
```
Comment (Pspritechologist) [inline Resources/Prototypes/SoundCollections/announcements.yml:4]: Pretty meh about this one, I'd personally say go without it.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]

File: Resources/ServerInfo/SimpleStation14/Guidebook/AlertProcedure/AlertProcedure.xml (hunk 5 above)
Comment (DEATHB4DEFEAT) [reply Resources/ServerInfo/SimpleStation14/Guidebook/AlertProcedure/AlertProcedure.xml:14]: I don't remember

File: Resources/Prototypes/SimpleStation14/Entities/Objects/Devices/Circuitboards/siliconchargers.yml (hunk 3 above)
Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/SimpleStation14/Entities/Objects/Devices/Circuitboards/siliconchargers.yml:31]: Lasers aren't real

File: Content.Server/StationEvents/Events/PowerGridCheckRule.cs
Comment (DEATHB4DEFEAT) [reply Content.Server/StationEvents/Events/PowerGridCheckRule.cs:1]: guh, none of the others were either

File: Resources/Textures/Corvax/Structures/Machines/VendingMachines/ammo.rsi/meta.json
Comment (DEATHB4DEFEAT) [reply Resources/Textures/Corvax/Structures/Machines/VendingMachines/ammo.rsi/meta.json:1]: this one's used

File: Content.Server/Shuttles/Systems/EmergencyShuttleSystem.cs [hunk 7]
```diff
@@ -188,15 +188,17 @@ public void CallEmergencyShuttle(EntityUid stationUid, StationEmergencyShuttleCo
         if (targetGrid == null)
         {
             _logger.Add(LogType.EmergencyShuttle, LogImpact.High, $"Emergency shuttle {ToPrettyString(stationUid)} unable to dock with station {ToPrettyString(stationUid)}");
+            // Parkstation-RandomAnnouncers Start
             // _chatSystem.DispatchStationAnnouncement(stationUid, Loc.GetString("emergency-shuttle-good-luck"), playDefaultSound: false);
-           // // TODO: Need filter extensions or something don't blame me.
-           // _audio.PlayGlobal("/Audio/Misc/notice1.ogg", Filter.Broadcast(), true);
+            // // TODO: Need filter extensions or something don't blame me.
+            // _audio.PlayGlobal("/Audio/Misc/notice1.ogg", Filter.Broadcast(), true);
```
Comment (Pspritechologist) [inline Content.Server/Shuttles/Systems/EmergencyShuttleSystem.cs:194]: Weird indent stuff?

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**